cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -371,6 +404,83 @@ def test_ssa_template(local_setup_provider, provider, soft_assert, vm_analysis_p
 
 
 @pytest.mark.rhv3
+@pytest.mark.tier(2)
+@pytest.mark.long_running
+def test_ssa_compliance(local_setup_provider, ssa_compliance_profile, ssa_profiled_vm, ssa_vm,
```

Comment:
Question: Don't understand why we need ssa_profiled_vm and ssa_vm they are the same